src/ExpressionEvaluator/Core/Test/FunctionResolver/VisualBasicParsingTests.cs

```diff
@@ -0,0 +1,470 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.VisualBasic;
+using Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator;
+using Microsoft.CodeAnalysis.VisualBasic.UnitTests;
+using Roslyn.Test.Utilities;
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator.UnitTests
+{
+    public class VisualBasicParsingTests : BasicTestBase
+    {
+        // Verify the set of keywords in the parser matches the VB compiler.
+        [Fact]
+        public void Keywords()
```

Comment:
The test shouldn't be necessary in C# because for C# we link in the `SyntaxFacts` type directly. Ideally we'd link in the VB `SyntaxFacts` type as well (so the set of keywords could not get out of sync between the compiler and breakpoint binder) but the VB `SyntaxFacts` is defined in VB whereas the breakpoint binder is C#.